DoctrineExtensions/Query/Search.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Requestum\ApiBundle\DoctrineExtensions\Query;
+
+use Doctrine\ORM\Query\AST\Functions\FunctionNode,
+    Doctrine\ORM\Query\Lexer;
+
+class Search extends FunctionNode
+{
+    public $fieldExpression = null;
+    public $likeExpression = null;
+
+    public function parse(\Doctrine\ORM\Query\Parser $parser)
+    {
+        $parser->match(Lexer::T_IDENTIFIER);
+        $parser->match(Lexer::T_OPEN_PARENTHESIS);
+        $this->fieldExpression = $parser->ArithmeticExpression();
+        $parser->match(Lexer::T_COMMA);
+        $this->likeExpression = $parser->StringPrimary();
+        $parser->match(Lexer::T_CLOSE_PARENTHESIS);
+    }
+
+    /**
+     * @example
+     * SEARCH(<field_name>,<like_value>) = true
+     *
+     * @param \Doctrine\ORM\Query\SqlWalker $sqlWalker
+     * @return string
+     */
+    public function getSql(\Doctrine\ORM\Query\SqlWalker $sqlWalker)
+    {
+        $params = $sqlWalker->getConnection()->getParams();
+        switch ($params['driver']) {
```

Comment:
According to the [doctrine configuration reference](https://www.doctrine-project.org/projects/doctrine-dbal/en/2.10/reference/configuration.html) `$params['driver']` can be even empty. It looks like we need to add a few lines to readme about the restrictions this pr brings